### ISSUE TITLE
Batch norm parameters

### ISSUE THREAD

**Issue by yaduydk97** [jiecaoyu/XNOR-Net-PyTorch#99]
![Capture1](https://user-images.githubusercontent.com/58845975/84002206-937e5380-a957-11ea-9e05-45dc7145df20.PNG)

I was trying to print all these parameters of batchnorm from your pretrained model, but I was able to print running mean and running variance only.
Is there some way to print beta and gamma (trainable parameters of batch norm)?

**Comment by jiecaoyu** [comment]
beta and gamma are weight and bias of batch norm layers.
